app/helpers/claims_helper.rb

```diff
@@ -11,14 +11,7 @@ def verified_fields(claim)
   end
 
   def eligibility_answers(eligibility)
-    [].tap do |a|
-      a << [t("questions.qts_award_year"), I18n.t("student_loans.questions.qts_award_years.#{eligibility.qts_award_year}"), "qts-year"]
-      a << [t("student_loans.questions.claim_school"), eligibility.claim_school_name, "claim-school"]
-      a << [t("questions.current_school"), eligibility.current_school_name, "still-teaching"]
-      a << [t("student_loans.questions.subjects_taught", school: eligibility.claim_school_name), subject_list(eligibility.subjects_taught), "subjects-taught"]
-      a << [t("student_loans.questions.leadership_position"), (eligibility.had_leadership_position? ? "Yes" : "No"), "leadership-position"]
-      a << [t("student_loans.questions.mostly_performed_leadership_duties"), (eligibility.mostly_performed_leadership_duties? ? "Yes" : "No"), "mostly-performed-leadership-duties"] if eligibility.had_leadership_position?
-    end
+    eligibility.class.module_parent::EligibilityAnswersPresenter.new(eligibility).answers
```

Comment:
I wonder if we should use `eligibility.claim.policy` here instead of `eligibility.class.module_parent`? I know under the hood that's what happens, but `claim.policy` feels like the API we've defined for identifying the policy namespace for a claim so the code may be more intention-revealing if we stick to using that?

Reply:
Makes sense - I'll change the helper's `#eligibility_answers` to take a claim instead of an eligibility.